page_objects/page_ovirt_hostedengine.py

```diff
@@ -17,26 +16,23 @@ class OvirtHostedEnginePage(PageTest):
     OVIRT_HOSTEDENGINE_FRAME_NAME = "/ovirt-dashboard"
     HOSTEDENGINE_LINK = "XPATH{}//a[@href='#/he']"
 
-    # Two start buttons
-    HE_START = "XPATH{}//span[@class='deployment-option-panel-container']/button[@class='btn btn-primary']"
-    HC_START = "XPATH{}//span[contains(@class, 'last-deployment-option-panel-container')]" \
-        "/button[@class='btn btn-primary']"
+    # Start button
+    HE_START = "XPATH{}//span[@class='deployment-option-panel-container']/button[text()='Start']"
 
     # Guide Links
-    GETTING_START_LINK = "XPATH{}//a[@href='https://ovirt.org/documentation/self-hosted/Self-Hosted_Engine_Guide/']"
-    MORE_INFORMATION_LINK = "XPATH{}//a[@href='http://www.ovirt.org']"
+    GETTING_START_LINK = "XPATH{}//a[contains(@href, 'Self-Hosted_Engine_Guide')]"
+    MORE_INFORMATION_LINK = "XPATH{}//a[contains(@href, 'www.ovirt.org')]"
```

Comment:
isn't there a link text? it's better to use the link text other than @href attribute